### ISSUE TITLE
Tried to use the option to get names for alliance ids and other things

### ISSUE THREAD

**Issue by Errious** [Blacksmoke16/GESI#8]
Hello,

after playing arround with alliance id function and my tries to get the names from them i run into an issue. 

For example, at this moment eve has 3009 alliance ids and the function can call them all, fine so far, but then when i try to get the names for them with the function =allianceNames() i have to insert a range, the problem is as soon the value of alliances increases over 3009 i get this error message [https://i.imgur.com/RKV3EOp.png](https://i.imgur.com/RKV3EOp.png)

Obviously the function can not identify the range from where to where an id is existing automaticly in the collumn, so i tried to use a workaround using `=ArrayFormula(MAX(IF(A3:A<>"";ROW(3:3500))))` to get the number of the last row with an alliance id and then my options are not much because i can not whrite the code to change the function that way it can identify the last cell in the collumn with an alliance id.

Would be nice to make it happen your script can do this automaticly and this for other functions too, to avoid the hassle to run into this kind of limitations or problems.

Next thing i saw is if i want to get the alliance corporations it returns only the corporation ids but not the names so i whrote this function in the way i am able to:
`
/**
 * Resolve a set of alliance corporation IDs to alliance corporation names
 * @param {boolean} opt_headers Default: True, Boolean if column headings should be listed or not.
 * @return Returns a list of id/name associations.
 * @customfunction
 */
function corporationNames(corporation_ids, opt_headers) {
    if (!corporation_ids) throw 'A range of corporation ids is required';
    return getArrayObjectResponse_(arguments.callee.name, '', opt_headers, {corporation_ids: corporation_ids});
}
`

This way i can get the names up to 100 corporations from the ids but as you remember the issue with calling the names for alliances the limmitations bringing the problem if an alliance has more then 100 corporations like goons or some others, so the function to load "Chunks" is needed.

Then it would be nice to be able to get the members from a corporation, this function is not existing at this moment, but would be greate to use for also hopefully soon introduced functions for the mining ledger esi calls to identify members and get some kind of statistics and the abillity to compare the lists from people who mined and get fetched with the observer fuction, i hope i explained this correct so u can understand.

So theese things are more QOL things then real issues but this can make the script more usable and user friendly.

Best Regards,

Samuel

**Comment by Blacksmoke16** [comment]
The issue with the alliance names was due to it not being able to parse an empty string to an integer as you prob figured out.  I added in a solution as well as the corporationNames endpoint.

ESI dev has been quite fast lately so i just been waiting for specific requests or until they reach XML parity.  Then i can just go thru and do some of the better ones.

I'll get the bug fix out and new moon mining stuff in next release.  Probably this weekend.
